src/lib/es5.d.ts

```diff
@@ -4382,35 +4398,60 @@ declare var Float64Array: Float64ArrayConstructor;
 /////////////////////////////
 
 declare namespace Intl {
+    type LocaleMatcherAlgorithm = "lookup" | "best fit";
+
+    interface SupportedLocalesOptions {
+        localeMatcher?: LocaleMatcherAlgorithm | undefined;
+    }
+
+    type CollatorOptionsUsage = "sort" | "search";
+    type CollatorOptionsCaseFirst = "upper" | "lower" | "false";
+    type CollatorOptionsSensitivity = "base" | "accent" | "case" | "variant";
+
     interface CollatorOptions {
-        usage?: "sort" | "search" | undefined;
-        localeMatcher?: "lookup" | "best fit" | undefined;
+        usage?: CollatorOptionsUsage | undefined;
+        localeMatcher?: LocaleMatcherAlgorithm | undefined;
         numeric?: boolean | undefined;
-        caseFirst?: "upper" | "lower" | "false" | undefined;
-        sensitivity?: "base" | "accent" | "case" | "variant" | undefined;
-        collation?: "big5han" | "compat" | "dict" | "direct" | "ducet" | "emoji" | "eor" | "gb2312" | "phonebk" | "phonetic" | "pinyin" | "reformed" | "searchjl" | "stroke" | "trad" | "unihan" | "zhuyin" | undefined;
+        caseFirst?: CollatorOptionsCaseFirst | undefined;
+        sensitivity?: CollatorOptionsSensitivity | undefined;
         ignorePunctuation?: boolean | undefined;
     }
 
     interface ResolvedCollatorOptions {
         locale: string;
-        usage: string;
-        sensitivity: string;
+        usage: CollatorOptionsUsage;
+        sensitivity: CollatorOptionsSensitivity;
         ignorePunctuation: boolean;
         collation: string;

```

Comment:
The resolved `collation` property remains here, even though the option to override it in `CollatorOptions` has moved to es2021.